src/client/components/sms-fraud/SendSMSMessageButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { ButtonHTMLAttributes, FunctionComponent } from 'react';
+import { TEST_IDS } from '../../testIDs';
+import styles from './smsVerificationFraud.module.scss';
+import { Alert } from '../common/Alert/Alert';
+import Button from '../common/Button/Button';
+import { SendMessageMutation } from '../../../pages/sms-fraud';
+
+type SendMessageButtonProps = {
+  sendMessageMutation: SendMessageMutation;
+  phoneNumber: string;
+  email: string;
+  type?: ButtonHTMLAttributes<HTMLButtonElement>['type'];
+};
+
+export const SendMessageButton: FunctionComponent<SendMessageButtonProps> = ({
+  sendMessageMutation,
+  phoneNumber,
+  email,
+  type,
+}) => {
+  const {
+    mutate: sendMessage,
+    data: sendMessageResponse,
+    error: sendMessageError,
+    isLoading: isLoadingSendSms,
+  } = sendMessageMutation;
+
+  return (
+    <>
+      {sendMessageError ? (
+        <Alert severity='error' className={styles.alert}>
+          {sendMessageError.message}
+        </Alert>
+      ) : null}
+      {sendMessageResponse ? (
+        <Alert severity={sendMessageResponse.severity} className={styles.alert}>
+          {sendMessageResponse.message}
+        </Alert>
+      ) : null}
+
+      <Button
+        disabled={isLoadingSendSms || sendMessageResponse?.data?.remainingAttempts === 0}
+        type={type}
```

Comment:
Why do we use different types for this button. Could we just use it as `submit` button in both cases? If would simplify this component.

Reply:
I am using the default HTML element form validation, so I need to use submit buttons to make that work AFAIK. The problem is that in the first step, the SendSMS button is the submit button (I need validation on the phone number input). But in the second step, the Verify button is the form submit button, so the SendSMS button cannot be.

Improved design proposal will likely come from Sabrina with some custom input elements for phone number or code, forcing us to abandon the native HTML form validation in favor of custom elements. Then we can make both buttons type "button" and simplify the component, I didn't want to spend time over-optimizing this now before I have the final design.